react/package.json

```diff
@@ -3,34 +3,30 @@
   "version": "0.0.0",
   "description": "Google Tag Manager.",
   "scripts": {
-    "test": "jest"
+    "test": "vtex-test-tools test"
   },
   "author": "VTEX",
   "license": "ISC",
   "devDependencies": {
-    "@types/node": "^10.12.18",
+    "@types/graphql": "^14.2.0",
+    "@types/jest": "^24.0.12",
+    "@types/node": "^12.0.0",
     "@types/ramda": "^0.25.42",
+    "@types/react": "^16.8.16",
+    "@types/react-intl": "^2.3.17",
+    "@vtex/test-tools": "^0.2.0",
+    "apollo-cache-inmemory": "^1.5.1",
+    "apollo-client": "^2.5.1",
     "identity-obj-proxy": "^3.0.0",
     "tslint": "^5.11.0",
     "tslint-config-vtex": "^2.0.0",
     "tslint-eslint-rules": "^5.4.0",
-    "typescript": "^3.1.1"
+    "typescript": "^3.4.5"
   },
   "jest": {
-    "verbose": true,
-    "transform": {
-      "^.+\\.(js|jsx|mjs)$": "<rootDir>/node_modules/babel-jest"
-    },
-    "transformIgnorePatterns": [
-      "[/\\\\]node_modules[/\\\\].+\\.(js|jsx|mjs)$"
-    ],
-    "setupFiles": [
+    "setupFilesAfterEnv": [
```

Comment:
Is this necessary? If so, what does it do? 😅

Reply:
Runs the `tests-setup.js` before running tests.